assets/js/blocks/featured-product/block.js

```diff
@@ -339,6 +341,30 @@ const FeaturedProduct = ( {
 											} )
 										}
 									/>
+									<TextareaControl
+										label={ __(
+											'Alt text (alternative text)',
+											'woo-gutenberg-products-block'
+										) }
+										value={ attributes.alt }
+										onChange={ ( newAlt ) => {
+											setAttributes( { alt: newAlt } );
+										} }
+										help={
+											<>
+												<ExternalLink href="https://www.w3.org/WAI/tutorials/images/decision-tree">
+													{ __(
+														'Describe the purpose of the image',
+														'woo-gutenberg-products-block'
+													) }
+												</ExternalLink>
+												{ __(
+													'Leave empty if the image is purely decorative.',
```

Comment:
Yes, agree, it makes more sense to me to go for the second option as well.

> We could even be a tad more sophisticated, and default alt-text with the short description only if the “Show description” option is turned off.
Not sure about that, I feel like if we do that we are adding some not obvious behavior without explaining it anywhere, and could be confusing for users.

Reply:
> Just realized that short_description can actually contain HTML, even images, so it might not be a good idea to use it as an alt. Should we go for the first option but without prefilling (or maybe prefilling with the product name), wdyt?

Interesting. Yea, then the first option seems the way to go, considering that it is also similar to how other blocks with ALT do it.

It's a bit sad we can't find a sensible default, because I know that most users won't even bother adding an ALT attribute. Maybe the default should just be the title, then? Or something like “Picture of PRODUCT_TITLE”?